docs/template_plugin/tests/functional/op_reference/clamp.cpp

```diff
@@ -0,0 +1,208 @@
+// Copyright (C) 2021 Intel Corporation
```

Comment:
please remove ngraph backend tests for these ops

Reply:
updated!